examples/doc-scan-identity-checks/src/controllers/index.controller.js

```diff
@@ -0,0 +1,70 @@
+const config = require('../../config');
+
+const {
+  DocScanClient,
+  SessionSpecificationBuilder,
+  SdkConfigBuilder,
+} = require('yoti');
+
+/**
+ * Create a Doc Scan session.
+ */
+async function createSession() {
+  const docScanClient = new DocScanClient(
+    config.YOTI_CLIENT_SDK_ID,
+    config.YOTI_PEM
+  );
+
+  const subject = {
+    subject_id: 'subject_id_string',
+  };
+
+  const identityProfileRequirements = {
+    trust_framework: 'UK_TFIDA',
+    scheme: {
+      type: 'DBS',
+      objective: 'BASIC',
+    },
+  };
+
+  const sessionSpec = new SessionSpecificationBuilder()
+    .withClientSessionTokenTtl(600)
+    .withResourcesTtl(90000)
+    .withUserTrackingId('some-user-tracking-id')
+    .withSubject(subject)
+    .withIdentityProfileRequirements(identityProfileRequirements)
+    .withSdkConfig(
+      new SdkConfigBuilder()
+        .withAllowsCameraAndUpload()
```

Comment:
Should we remove that or is it mandatory (I know the API wants it, but I can not remember if it needs to be specified)?

Reply:
Actually, lemme check with Pawel and find out what needs to be specified and what not, and accordingly add/remove them. I am also wondering about the mobile handoff.

Reply:
Updated the example configuration to:

```  const sessionSpec = new SessionSpecificationBuilder()
    .withClientSessionTokenTtl(600)
    .withResourcesTtl(90000)
    .withUserTrackingId('some-user-tracking-id')
    .withSubject(subject)
    .withIdentityProfileRequirements(identityProfileRequirements)
    .withSdkConfig(
      new SdkConfigBuilder()
        .withPrimaryColour('#2d9fff')
        .withLocale('en-GB')
        .withPresetIssuingCountry('GBR')
        .withSuccessUrl(`${config.YOTI_APP_BASE_URL}/success`)
        .withErrorUrl(`${config.YOTI_APP_BASE_URL}/error`)
        .build()
    )
    .build();```

Reply:
Tested it in prod with this config and I was able to create a session successfully.